src/code/padmgr.c

```diff
@@ -44,132 +56,133 @@ void PadMgr_UnlockReleaseControllerQueue(PadMgr* padmgr, OSMesgQueue* ctrlrqueue
                         }
                     }
                 }
             } else {
-                if (padmgr->unk_2AE[i] != 0) {
-                    if (padmgr->unk_2AE[i] == 1) {
+                if (padmgr->pak_type[i] != 0) {
+                    if (padmgr->pak_type[i] == 1) {
                         osSyncPrintf(VT_FGCOL(YELLOW));
                         osSyncPrintf("padmgr: %dコン: %s\n", i + 1, "振動パックが抜かれたようです");
                         osSyncPrintf(VT_RST);
-                        padmgr->unk_2AE[i] = 0;
+                        padmgr->pak_type[i] = 0;
                     } else {
                         osSyncPrintf(VT_FGCOL(YELLOW));
                         osSyncPrintf("padmgr: %dコン: %s\n", i + 1,
                                      "振動パックではないコントローラパックが抜かれたようです");
                         osSyncPrintf(VT_RST);
-                        padmgr->unk_2AE[i] = 0;
+                        padmgr->pak_type[i] = 0;
                     }
                 }
             }
         }
     }
 
-    if (!var1) {
-        var3 = D_8016A4F0 % 4;
+    if (!tried_rumble_comm) {
+        ctrlr = D_8016A4F0 % 4;
 
-        if ((padmgr->unk_2AA[var3] != 0) && (padmgr->pad_status[var3].status & 1) && (padmgr->unk_2AE[var3] != 1)) {
-            var4 = osProbeVibrationPack(ctrlrqueue, &padmgr->unk_controller[var3], var3);
+        if ((padmgr->Key_switch[ctrlr] != 0) && (padmgr->pad_status[ctrlr].status & 1) && (padmgr->pak_type[ctrlr] != 1)) {
+            var4 = osProbeVibrationPack(ctrlrqueue, &padmgr->unk_controller[ctrlr], ctrlr);
 
             if (var4 == 0) {
-                padmgr->unk_2AE[var3] = 1;
-                osSetVibration(&padmgr->unk_controller[var3], 1);
-                osSetVibration(&padmgr->unk_controller[var3], 0);
+                padmgr->pak_type[ctrlr] = 1;
+                osSetVibration(&padmgr->unk_controller[ctrlr], 1);
+                osSetVibration(&padmgr->unk_controller[ctrlr], 0);
                 osSyncPrintf(VT_FGCOL(YELLOW));
-                osSyncPrintf("padmgr: %dコン: %s\n", var3 + 1, "振動パックを認識しました");
+                osSyncPrintf("padmgr: %dコン: %s\n", ctrlr + 1, "振動パックを認識しました");
                 osSyncPrintf(VT_RST);
             } else if (var4 == 11) {
-                padmgr->unk_2AE[var3] = 2;
+                padmgr->pak_type[ctrlr] = 2;
             } else if (var4 == 4) {
-                LOG_NUM("++errcnt", ++D_8012D284, "../padmgr.c", 282);
+                LOG_NUM("++errcnt", ++errcnt, "../padmgr.c", 282);
                 osSyncPrintf(VT_FGCOL(YELLOW));
-                osSyncPrintf("padmgr: %dコン: %s\n", var3 + 1, "コントローラパックの通信エラー");
+                osSyncPrintf("padmgr: %dコン: %s\n", ctrlr + 1, "コントローラパックの通信エラー");
                 osSyncPrintf(VT_RST);
             }
         }
     }
 
     D_8016A4F0++;
-    PadMgr_UnlockReleaseControllerQueue(padmgr, ctrlrqueue);
+    padmgr_UnlockSerialMesgQ(padmgr, ctrlrqueue);
 }
 #else
-u32 D_8012D284 = 0;
+u32 D_8012D284 = 0; //errcnt
```

Comment:
we want to put a space after // for comments. I believe clang would take care of this for you so make sure to run format.sh before submitting pr's